crates/uv-scripts/src/lib.rs

```diff
@@ -95,6 +95,39 @@ impl Pep723Script {
         })
     }
 
+    pub async fn create_new_script(
```

Comment:
Lets call this `create` and rename the method above to `init`?

Reply:
Yes sounds great, done